src/deploy/002_settlement.ts

```diff
@@ -15,7 +15,7 @@ const deploySettlement: DeployFunction = async function ({
 
   await deploy(settlement, {
     from: deployer,
-    gasLimit: 3000000,
+    gasLimit: 3269439,
```

Comment:
nit: instead of using an exact value, can we just make this:
```suggestion
    gasLimit: 3.5e6,
```

Reply:
Yea I could, although this is related to an issue I opened with hardhat and I told them they could reduce this value by any number to reproduce the failure. I will go edit the instructions on how to reproduce